resources/scss/ceres/views/ItemList/_loading-animation.scss

```diff
@@ -26,6 +26,7 @@
   height: 100%;
   width: 18px;
   display: inline-block;
+  margin: 0 .1rem;
```

Comment:
Den Fix für die loading animation sollten wir nochmal separat in beta machen

Reply:
done